Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -517,6 +519,7 @@ defineSuite([
 
         return Cesium3DTilesTester.loadTileset(scene, pointCloudNoColorUrl).then(function(tileset) {
             tileset.pointCloudShading.eyeDomeLighting = false;
+            tileset.refine = Cesium3DTileRefine.REPLACE;
```

Comment:
The test is still failing. Use `tileset.root` to get the root tile.

Reply:
My bad, when I did `fit` I only did it on the first guy, didn't see that there were more below